design/sketch.webidl

```diff
@@ -147,6 +147,12 @@ enum GPUTextureFormat {
     "rgba32uint",
     "rgba32sint",
     "rgba32float",
+    /* Depth and Stencil formats */
+    "depth16unorm",
+    "depth32float",
+    "stencil8",
```

Comment:
Unless I am missing something, D3D12 does not natively support 'stencil8'. 

The core standard should only contain formats supported by all APIs.

Reply:
Updated, thanks! The rest look good on the Vk side, from my understanding.